doc/tutorials/content/sources/gpu_octree/CMakeLists.txt

```diff
@@ -0,0 +1,19 @@
+cmake_minimum_required(VERSION 2.8 FATAL_ERROR)
+
+project(gpu_octree)
+
+find_package(PCL 1.12 REQUIRED)
+
+include_directories(${PCL_INCLUDE_DIRS})
+link_directories(${PCL_LIBRARY_DIRS})
+add_definitions(${PCL_DEFINITIONS})
+
+#Searching CUDA
+find_package(CUDA)
+
+#Include the FindCUDA script
+include(FindCUDA)
+
+cuda_add_executable (gpu_octree gpu_octree.cpp)
+target_link_libraries (gpu_octree ${PCL_LIBRARIES})
+
```

Comment:
BTW, the PCL 1.12 requirement is actually the cause of the current CI error, since obviously 1.12 is not there yet.
```shell
CMake Error at CMakeLists.txt:5 (find_package):
  Could not find a configuration file for package "PCL" that is compatible
  with requested version "1.12"
```
How do I get around this, since the API used isn't available in 1.11?

Reply:
Request version 1.11.1.99. This will get it passing.

Reply:
The tutorial is build upon 16.04 image without CUDA isn't it? So it will probably fail again.

Reply:
Make the transition to the 18.04 build
https://github.com/PointCloudLibrary/pcl/blob/f46f8a93462282d45ff6c87f0c07e6c1be15edb1/.ci/azure-pipelines/tutorials.yaml#L1-L8

Reply:
@kunaltyagi @shrijitsingh99 I need some know-how assistance. What are the required steps here to transition the tutorials job to have access to this build environment?

https://github.com/PointCloudLibrary/pcl/blob/f46f8a93462282d45ff6c87f0c07e6c1be15edb1/.ci/azure-pipelines/env.yml#L48-L53

Reply:
Do you mean to share the variables in the tutorial jobs too? or define these variables for the tutorial job as well.

Reply:
Just make tutorials run on something with cuda support :smile:

Reply:
https://github.com/PointCloudLibrary/pcl/blob/f46f8a93462282d45ff6c87f0c07e6c1be15edb1/.ci/azure-pipelines/env.yml#L45
Setting this true will enable CUDA for 16.04 docker image and push that image to docker hub once the change is merged with master. After that tutorial will have CUDA support.

The side effect is that all the 16.04 builds will now run on a container that has CUDA support.
I don't see any issue because of this as of now.